packages/components/src/combobox-control/stories/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ComboboxControl from '../';
+
+export default { title: 'ComboboxControl', component: ComboboxControl };
+
+const options = [
+	{
+		key: 'small',
+		name: 'Small',
+		style: { fontSize: '50%' },
+	},
+	{
+		key: 'normal',
+		name: 'Normal',
+		style: { fontSize: '100%' },
+	},
+	{
+		key: 'large',
+		name: 'Large',
+		style: { fontSize: '200%' },
```

Comment:
What would that test? I'm going with the default theme's values here.

Reply:
To test filtering with multiple results. I had to add a few more items locally to do that, so thought it might be useful to just have them out of the box.